.gitignore

```diff
@@ -15,6 +15,7 @@ pids
 libs/
 node_modules/
 bower_components/
+*.js.map
```

Comment:
This one is because IntelliJ has a TypeScript plugin that creates JS map files automatically (to ease the debugging), but I don't think we want those on the SCM.